### PR TITLE
Change the menu option name from "Liberty: View effective POM" to "Liberty: View pom.xml"

### LINKED ISSUE #615: The "Liberty: view effective POM" menu option is misnamed
The "effective POM" is what's generated by incorporating defs in the parent (inheritance) plus applying profiles... you can see it with `mvn help:effective-pom` or with the Maven Tool Window.

We're just showing the pom.xml file.  A name like `Liberty: view pom.xml` would fit bitter.

### PR DESCRIPTION
**Pull request by vaisakhkannan**
Fixes #615 --> Changed the menu option name from "Liberty: View effective POM" to "Liberty: View pom.xml"

### CODE REVIEW

**Review by anusreelakshmi934 — COMMENTED** [review]
Update the copyrights in modified files.

**Review by anusreelakshmi934 — APPROVED** [review]

File: src/main/resources/messages/LibertyBundles.properties [hunk 1]
```diff
@@ -25,7 +25,7 @@ action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.text=Liberty: Vi
 action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.description=Open the Maven Surefire test report for the corresponding Liberty project
 action.io.openliberty.tools.intellij.actions.ViewTestReport.text=Liberty: View test report
 action.io.openliberty.tools.intellij.actions.ViewTestReport.description=Open the Gradle test report for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.ViewEffectivePom.text=Liberty: View effective POM
+action.io.openliberty.tools.intellij.actions.ViewEffectivePom.text=Liberty: View pom.xml
```
Comment (TrevCraw) [inline src/main/resources/messages/LibertyBundles.properties:28]: We should also update the name of the action here and throughout the rest of the code as well. `ViewPom` or `ViewPomXml` would be good.

**Review by TrevCraw — CHANGES_REQUESTED** [review]

**Review by anusreelakshmi934 — DISMISSED** [review]

**Review by TrevCraw — CHANGES_REQUESTED** [review]
The following lines also need to be updated:
- LibertyBundles.properties - https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/src/main/resources/messages/LibertyBundles.properties#L93-L94
- ViewPomXml.java - https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/src/main/java/io/openliberty/tools/intellij/actions/ViewEffectivePom.java#L30

**Review by anusreelakshmi934 — APPROVED** [review]
Looks good to me.

File: src/main/resources/messages/LibertyBundles.properties [hunk 2]
```diff
@@ -91,7 +91,7 @@ run.tests.liberty.dev=run tests
 run.tests.dev=Run tests
 
 # View effective pom action
```
Comment (TrevCraw) [inline src/main/resources/messages/LibertyBundles.properties:93]:
```suggestion
# View pom.xml action
```

**Review by TrevCraw — CHANGES_REQUESTED** [review]

**Review by anusreelakshmi934 — APPROVED** [review]

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**